ZitiNativeApiForDotnetCore/library/ziti4dotnet.c

```diff
@@ -86,4 +88,67 @@ int ziti_context_event_status(const ziti_event_t* e) {
     else {
         return 0;
     }
-}
\ No newline at end of file
+}
+
+typedef int (*printer)(void* arg, const char* fmt, ...);
+
+static int ziti_dump_to_log_op(void* stringsBuilder, const char* fmt, ...) {
+    static char line[4096];
+
+    va_list vargs;
+    va_start(vargs, fmt);
+    vsnprintf(line, sizeof(line), fmt, vargs);
+    va_end(vargs);
+
+    if (strlen(stringsBuilder) + strlen(line) > MAXBUFFERLEN) {
+        return -1;

```

Comment:
why not just append what you can in this case? and if you make the buffer huge hopefully this won't be an issue